authors/apps/authentication/serializers.py

```diff
@@ -1,10 +1,12 @@
 import re
 
-from django.contrib.auth import authenticate
 from django.core.validators import RegexValidator, ValidationError
-from rest_framework import serializers
+from authors.apps.profiles.serializers import GetProfileSerializer
+from .models import User, PasswordReset, UserNotification
 from rest_framework.validators import UniqueValidator
-
+from notifications.models import Notification
```

Comment:
Is this the correct path?

Reply:
Yes ... I think what might be missing from your setup to make this not work is the package `django-notifs`. I haven't been able to figure out why but pipenv always seems to have a new problem. So I manually installed the packages locally using pip. I'll commit the requirements.txt file along with the requested changes.

Reply:
Our `Pipfile.lock` may be corrupted, you may have to delete it; and `pipenv  install` the new packages so a new piffle.lock can be created as well as update the `Pipfile`